packages/core/src/components/Picker/Picker.tsx

```diff
@@ -60,39 +122,311 @@ const Picker: React.FC<Props> = ({
     }
   }, [defaultValue]);
 
-  const onValueChange = React.useCallback(
-    (itemValue: string, itemIndex: number) => {
-      if (placeholder && itemIndex === 0) {
-        return;
-      }
-      onValueChangeOverride &&
-        onValueChangeOverride(String(itemValue), itemIndex);
-    },
-    [placeholder, onValueChangeOverride]
-  );
-
   const normalizedOptions = normalizeOptions(options);
 
   const pickerOptions = placeholder
     ? [{ value: placeholder, label: placeholder }, ...normalizedOptions]
     : normalizedOptions;
 
+  const { viewStyles, textStyles } = extractStyles(style);
+
+  const { colors } = theme;
+
+  const borders = [
```

Comment:
this seems like something that should be extracted out similar to `extractStyles` just in case we need it somewhere else - wdyt?

Reply:
@peterpme Done.